livy/src/main/resources/interpreter-setting.json

```diff
@@ -118,6 +118,11 @@
         "defaultValue": "1000",
         "description": "Max number of Spark SQL result to display."
       },
+      "zeppelin.livy.spark.sql.truncate": {
+        "propertyName": "zeppelin.livy.spark.sql.truncate",
+        "defaultValue": "true",
+        "description": "If true, truncate strings greater than 20 characters."
```

Comment:
btw, I think perhaps this could be more useful if this is the truncate line numbers (eg. 20), instead of true/false.
I think I recall the Spark interpreter has it that way too.